optuna_dashboard/_preference_setting.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from optuna.storages import BaseStorage
+
+from .preferential._study import PreferentialStudy
+
+
+if TYPE_CHECKING:
+    from typing import Literal
+
+    OUTPUT_COMPONENT_TYPE = Literal["Note", "Artifact"]
```

Comment:
How about using lower-case, to be consistent with "minimum"/"maximum" used in optuna?